compiler/src/iree/compiler/Codegen/LLVMCPU/DispatchABI.cpp

```diff
@@ -380,13 +378,12 @@ LLVM::LLVMStructType HALDispatchABI::getDispatchStateType(
       context, "iree_hal_executable_dispatch_state_v0_t");
   if (structType.isInitialized()) return structType;
 
-  auto indexType = typeConverter->convertType(IndexType::get(context));
-  auto int8Type = IntegerType::get(context, 8);
+  // auto indexType = typeConverter->convertType(IndexType::get(context));
```

Comment:
Just delete the statements instead of commenting them out.

Reply:
Done!